solores/Cargo.toml

```diff
@@ -2,8 +2,8 @@
 name = "solores"
 description = "Solana IDL to Rust client / CPI interface generator"
 license = "MIT OR Apache-2.0"
-repository = "https://github.com/igneous-labs/solores"
-version = "0.7.0"
+repository = "https://github.com/vybenetwork/solores"
```

Comment:
this one's probably not necessary for upstream PR :)

Reply:
Reverted, thanks.